index.html

```diff
@@ -29,5 +29,31 @@ <h1 class="logo">Loruki</h1>
         </nav>
       </div>
     </div>
+    <br />
+    <!-- footer -->
+    <footer>
+      <div>
+        <h1>Loruki</h1>
+        <p>Copyright &copy; 2022</p>
+      </div>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="features.html">Features</a></li>
+          <li><a href="docs.html">Docs</a></li>
+        </ul>
+      </nav>
+      <div class="social">
+        <a
+          href="https://github.com/lab-brussels-1/loruki-group-2"
+          target="_blank"
+          alt="github page"
+          ><i class="fab fa-github fa-2x"></i
+        ></a>
+        <a href="#"><i class="fab fa-facebook fa-2x"></i></a>
```

Comment:
@Shewitttt you need to put some text between the opening and closing a tag..
so for example:

```
<a href="#">SOMETHING<i class="fab fa-facebook fa-2x"></i></a>